htdocs/install/page_dbconnection.php

```diff
@@ -45,37 +45,59 @@
 }
 
 $error = '';
-if ($_SERVER['REQUEST_METHOD'] == 'POST' && !empty( $vars['DB_HOST'] ) && !empty( $vars['DB_USER'] )) {
-	$func_connect = empty( $vars['DB_PCONNECT'] ) ? "mysql_connect" : "mysql_pconnect";
-	if (! ( $link = @$func_connect( $vars['DB_HOST'], $vars['DB_USER'], $vars['DB_PASS'], true ) )) {
-		$error = ERR_NO_DBCONNECTION;
+if ($_SERVER['REQUEST_METHOD'] == 'POST' && !empty($vars['DB_HOST']) && !empty($vars['DB_USER'])) {
+	switch ($vars['DB_TYPE']) {
+		case 'mysql':
```

Comment:
again. should we care about old mysql drivers that doesn't work on unsupported PHP version?